packages/next/src/build/output/index.ts

```diff
@@ -225,14 +237,22 @@ export function ampValidation(
 export function watchCompilers(
   client: webpack.Compiler,
   server: webpack.Compiler,
-  edgeServer: webpack.Compiler
+  edgeServer: webpack.Compiler,
+  logging?:
```

Comment:
we don't need to pass it down into build store. the logging can be a boolean value saved in console store, and when `logging: boolean` is false, skip logging

Reply:
Removed it! But not sure how the `logging` value be passed on webpack HMR unless passing to that `watchCompilers`.

Reply:
if next config is changed, the server will be restarted